gulpfile.js

```diff
@@ -0,0 +1,44 @@
+const gulp			= require('gulp');
```

Comment:
In fact, is it a js convention to add tabspaces before =? Otherwise we can just make it the usual var declarations:

const gulp = require('gulp');
const babel = require('gulp-babel');
const watch = require('gulp-nodemon');

..etc.